clicache/integration-test/ThinClientDeltaTestN.cs

```diff
@@ -832,50 +832,55 @@ void runExpirationWithDelta()
 
     //#region Tests
 
-    [Test]
+    //[Test]
     public void PutWithDeltaAD()
     {
       runDeltaWithAppdomian(false);
-      runDeltaWithAppdomian(true);//cloning enable
     }
 
-    [Test]
+    //hangs [Test]
+    public void PutWithDeltaADWithCloning()
+    {
+      runDeltaWithAppdomian(true);
+    }
+
+    //passed [Test]
```

Comment:
like this one that says passed but is commented out?

Reply:
Grrrfff... Will fix.

Reply:
There's a few of those in this file, will let you go through them

Reply:
Odd, I thought I checked in the uncommented version. Will double check.

Reply:
Doh! Still on my machine.